_data-prepper/pipelines/configuration/sources/dynamo-db.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+title: dynamodb
+parent: Sources
+grand_parent: Pipelines
+nav_order: 3
+---
+
+# dynamodb
+
+The `dynamodb` source enables change data capture (CDC) on [DynamoDB](https://aws.amazon.com/dynamodb/) tables. It can receive table events such as `create`, `update`, or `delete` using DynamoDB streams and supports initial snapshots using [point-in-time recovery (PITR)](https://aws.amazon.com/dynamodb/pitr/).
+
+The source includes two ingestion options to stream DynamoDB events:
+
+1. _Full Initial Snapshot_ using [point-in-time recovery (PITR)](https://aws.amazon.com/dynamodb/pitr/) gets an initial snapshot of the current state of the DynamoDB table. This requires the PITR Snapshots and DyanmoDB option enabled on your DynamoDB table.
+2.  Stream events from DynamoDB streams without full initial snapshots. This is useful for users who already have a mechanism for snapshots within their pipelines. This requires that DynamoDB stream is enabled on the DynamoDB table.
+
+## Usage
+
+The following example pipeline specifies DynamoDB as a source. It ingests data from a DyanmoDB table, named `table-a` through a PITR snapshot. It also indicates the `start_position`, which tells the pipeline how to read events DynamoDB stream. 
+
+```json
+version: "2"
+cdc-pipeline:
+  source:
+    dynamodb:
+      tables:
+        - table_arn: "arn:aws:dynamodb:us-west-2:123456789012:table/table-a"
+          export:
+            s3_bucket: "test-bucket"
+            s3_prefix: "xxx/"
+          stream:
+            start_position: "LATEST" # Read latest data from streams (Default)
+      aws:
+        region: "us-west-2"
+        sts_role_arn: "role-arn"
+```
+
+## Configuration options
+
+The following tables describe the configuration options for the `dynamodb` source.
+
+
+The following settings are shared between the `dynamodb` source.
+
+Option | Required | Type | Description
+:--- | :--- | :--- | :---
+`aws` | Yes | AWS | The AWS configuration. See [aws](#aws) for more information.
+`acknowledgments` | No | Boolean  | When `true`, enables `s3` sources to receive [end-to-end acknowledgments]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/pipelines#end-to-end-acknowledgments) when events are received by OpenSearch sinks.
+`shared_acknowledgement_timeout` | No | Duration | The time before the data read from a DynamoDB stream expires when used with acknowledgements. Default is 10 minutes.
+`s3_data_file_acknowledgment_timeout` | No | Duration | The time before the data read from a DynamoDB export expires when used with acknowledgments. Default is 10 minutes.
+`tables` | Yes | Object | The configuration for the DynamoDB table. See [tables](#tables) for more information.
```

Comment:
Rather than `Object` in the third column, let's use either `List of Table` or `List`.